src/main/resources/edu/harvard/iq/dataverse/codebook2-0.xsl

```diff
@@ -0,0 +1,2538 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I just wanted to remind everyone that there's a related codebook file at `src/test/java/edu/harvard/iq/dataverse/export/ddi/codebook.xsd`